src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareUtils.cs

```diff
@@ -43,16 +43,34 @@ internal static DacDeployOptions CreateSchemaCompareOptions(DeploymentOptions de
                 foreach (PropertyInfo deployOptionsProp in deploymentOptionsProperties)
                 {
                     var prop = propType.GetProperty(deployOptionsProp.Name);
-                    if (prop != null)
+                    // Set the excludeObjectTypes values to the DacDeployOptions
+                    if (prop != null && deployOptionsProp.Name == nameof(deploymentOptions.ExcludeObjectTypes))
                     {
+                        List<ObjectType> finalExcludeObjects = new List<ObjectType> { };
                         var val = deployOptionsProp.GetValue(deploymentOptions);
-                        var selectedVal = val.GetType().GetProperty("Value").GetValue(val);
+                        string[] excludeObjectTypeOptionsArray = (string[])val.GetType().GetProperty("Value").GetValue(val);
 
-                        // Set the excludeObjectTypes values to the DacDeployOptions
-                        if (selectedVal != null && deployOptionsProp.Name == nameof(deploymentOptions.ExcludeObjectTypes))
+                        if (excludeObjectTypeOptionsArray != null)
                         {
-                            prop.SetValue(dacOptions, selectedVal);
+                            foreach(string objectTypeValue in excludeObjectTypeOptionsArray)
+                            {
+                                // Convert the first char of the enum to upper case, as the excludeObjectTypes values are coming in lower case which needs to be converted
+                                string name = char.ToUpper(objectTypeValue.ToCharArray()[0]) + objectTypeValue.Substring(1);
+                                ObjectType enumName = new ObjectType(); 
+                                   
+                                if (name != null && Enum.TryParse(name, out enumName))
```

Comment:
Better yet, `Enum.TryParse(name, ignoreCase:true, out ObjectType enumName)` so avoid the name conversion entirely.  Also, note the inline declaration of the ObjectType instead of having to do that separately.

Reply:
Also, is "enumName" the best name for that var?  Maybe just "objectType" would be better, but you know the code better :)

Reply:
Yes, not required with the ignoreCase! Thanks